__DEFINES/obj_defines.dm

```diff
@@ -0,0 +1,32 @@
+//Quality
+
+#define AWFUL 0
+#define SHODDY 1
+#define POOR 2
+#define NORMAL 3
+#define GOOD 4
+#define SUPERIOR 5
+#define EXCELLENT 6
+#define MASTERWORK 7
+#define LEGENDARY 8
+
+/proc/getQualityString(var/quality)
+	switch(quality)
+		if(0)
```

Comment:
you literally defined the defines above you nerd.
Also in my opinion I'd make it an assoc list ie
quality_to_string = list("[AWFUL]" ="awful","[SHODDY]" = "shoddy") etc so you could just return quality_to_string[quality]